code/__DEFINES/logging.dm

```diff
@@ -29,15 +29,16 @@
 #define LOG_CHAT			(1 << 6)
 #define LOG_COMMENT			(1 << 7)
 #define LOG_TELECOMMS		(1 << 8)
-#define LOG_OOC				(1 << 9)
-#define LOG_ADMIN			(1 << 10)
-#define LOG_OWNERSHIP		(1 << 11)
-#define LOG_GAME			(1 << 12)
-#define LOG_ADMIN_PRIVATE	(1 << 13)
-#define LOG_ASAY			(1 << 14)
-#define LOG_MECHA			(1 << 15)
-#define LOG_VIRUS			(1 << 16)
-#define LOG_CLONING			(1 << 17)
+#define LOG_NTSL			(1 << 9)
```

Comment:
Why don't you make this (1<< 18)

Reply:
because I wanted it to be directly below LOG_TELECOMMS, since they're thematically linked